showcase/firebase.md

```diff
@@ -40,6 +81,15 @@ Firestore is the next generation of Realtime Database, it has better queries and
 
 You can find all the products offered by Firebase [here]( https://firebase.google.com/products/ ).
 
+Firebase can be added in two ways in a web project, you can add it as a script using a google CDN  or you can install it using npm( [Firebase](https://www.npmjs.com/package/firebase) ).
+
+```
+npm install --save firebase
+```
+One problem with firebase npm package is that is quite big and will increase your bundle size.
```

Comment:
"with **the** npm package is that **it** is quite big" - sounds better in this context

Reply:
done